server/services/accountService.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/coinbase/rosetta-sdk-go/types"
 )
 
+type accountWithBalance struct {
```

Comment:
I don't think this is the best naming, since the account seems to have a nonce and blockCoordinates as well

Reply:
Renamed.